content/docs/practices/naming.md

```diff
@@ -61,3 +61,22 @@ pairs represents a new time series, which can dramatically increase the amount
 of data stored. Do not use labels to store dimensions with high cardinality
 (many different label values), such as user IDs, email addresses, or other
 unbounded sets of values.
+
+
+## Base units
+
+Prometheus does not have any units hard coded. For better compability, base
+units should be used. The following lists metrics families with their base unit:
+
+| Family | Base unit | Remark | 
+| -------| --------- | ------ |
+| Time   | seconds   |        |
+| Temperature | celsius |     |
+| Length | meter | |
+| Bytes  | bytes | | 
+| Bits   | bytes | |
+| Percent | percent | Values are 0-1 |
+| Voltage | volt | |
+| Electric current | ampere | | 
+| Power | watts | |
```

Comment:
Watt is Joule per second. Watt is a unit of Power, while Joule is a unit of Energy.

We can also just leave it out.